src/common/responses.py

```diff
@@ -108,7 +109,7 @@ async def wrapper_decorator(*args, **kwargs) -> TResponse | JSONResponse:
             except BadRequestException as e:
                 if logger.level <= logging.DEBUG:
                     traceback.print_exc()
-                logger.error(e)
+                logger.error(e, extra={"Traceback": get_traceback()})
```

Comment:
IMO, these are not errors. They are discovered, handled, and known, execution aborts based on "BadInput". If you still want to see that caused that, it should be logged as "DEBUG". 

Same goes for the rest of the changes in this file

Reply:
👍  the plan is to go through all log statements after we have started logging to see what we would like to log, so seperate PR